src/InvokePlaster.ps1

```diff
@@ -1133,6 +1133,8 @@ function Invoke-Plaster {
 
                         # Eval script expression delimiters
                         if ($content -and ($content.Count -gt 0)) {
+                            # Sets the Variable to the currently used template
+                            Set-Variable -Name PLASTER_TemplateSource -Value $srcPath -Scope Script
```

Comment:
I think this is reasonable but I'd probably want to name this something like `PLASTER_TemplateFileSource`.

Reply:
In my tests, it worked fine... 
`PLASTER_TemplateFileSource` is more precise than `PLASTER_TemplateSource`

I was not sure if there are situations, where Files are created just by code (without any template file) and then, `PLASTER_TemplateSource` could be used for both situations (Generated by code and generated by Template)